tests/Feature/Authentication/LoginTest.php

```diff
@@ -27,6 +28,39 @@ public function testLogsFailedLoginAttempt()
         ]);
     }
 
+
+    public function testLoginThrottleConfigIsRespected()
+    {
+
+        // Why do we need this? The app should already be set up
```

Comment:
We don't run seeders in tests so there is no user yet.

Reply:
Should we add a user to the Settings support stuff? That test kicks you back to `/setup` if we don't create that pointless user for that test.

Reply:
I'm ok with having to make pointless users every once in a while in tests since it doesn't happen often. Most of the time we're making a user to act as so it doesn't pop up.